src/main/java/decodex/parser/Parser.java

```diff
@@ -67,6 +70,28 @@ public String getCommandType(String userInput) throws ParserException {
         return Arrays.copyOfRange(tokens, STARTING_ARGUMENTS_INDEX, tokens.length);
     }
 
+    /**
+     * Returns the Argument portion of the user input as a whole string.
+     *
+     * @param userInput The input specified by the user.
+     * @return The argument portion of the user input as an array.
+     */
+    public String getUserArgumentsAsString(String userInput) throws ParserException {
+        String[] tokens = userInput.split(" ");

```

Comment:
Perhaps consider doing a "left strip" before `split()`? 

Also perhaps add a `-1` to make it into `split(" ", -1")`? This will ensure that even empty tokens will be kept. This is important as splitting this `"hello  world"` (note that double space in the middle) and then joining with space will result in `"hello world"` (only single space in the middle). But adding the `-1` will allow us to retain that double space.